index.html

```diff
@@ -51,9 +62,11 @@ <h3>Let's Go For A Walk</h3>
       
         <p>Suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth.</p>
       
-        <div class="btn">
-          Sign Up
-        </div>
+        <a href="sign-up.html">
+          <div class="btn">
+            Sign Up
```

Comment:
Interesting technique to create a button here. I love seeing that you know you can wrap other elements in `a` tags. You could also do this with a regular `button` if you wanted to simplify styling :)